src/baguetteBox.js

```diff
@@ -69,6 +69,14 @@
     var data = {};
     // Array containing temporary images DOM elements
     var imagesElements = [];
+    // Array containing temporary images figcaption IDs for aria-labelledby property
+    var imagesCaptionsIds = [];
+    // Array containing temporary images figcaption IDs for aria-describedby property
+    var imagesFiguresIds = [];
+    // variable that will keep track of the last focused element before opening the overlay
+    var lastFocus = '';
+    // variable used to track status of overlay
+    var overlayStatus = false;
```

Comment:
The meaning of this variable is very obscure to me. Is it about detecting whether the overlay is visible? In such case why not check against `overlay.style.display === 'block'`?